modules/generator/processor/spanmetrics/config.go

```diff
@@ -67,3 +70,25 @@ func (ic *IntrinsicDimensions) ApplyFromMap(dimensions map[string]bool) error {
 	}
 	return nil
 }
+
+type FilterPolicy struct {
```

Comment:
I think we're missing one layer here. We want to do a slice of include/exclude policies right?

```
{
    include: [
        {
            match_type: 'strict'
            attributes: [
                {
                    key: 'resource.service.name',
                    value: 'test-service',
                },
                {
                    key: 'span.status',
                    value: 'OK',
                }
            ],
        },
        {
            match_type: 'strict'
            attributes: [
                {
                    key: 'span.name',
                    value: 'Read',
                },
            ],
        },
    ],
    exclude: [],
}
```

where the attributes are "or"s and the policies are "and"s:
So the above would read: I want only Read spans that are either from "test-service" or with a status of "OK"

Reply:
That was my original thinking too, but then I read closer in the original issue that pointed upstream to what OTEL is doing.

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/internal/filter/filterconfig/config.go#L32

Double check me there, but I think we're doing what is described in that godoc.

For the example above, the same can be achieved by including multiple policies with each their own `include`.

I'd initially been thinking of a different approach to polices that was a little more flexible, but I couldn't come up with a scenario where the same results couldn't have been achieved with the otel config example.  Keeping some alignment seems okay to me, but the goofy part is that we're keeping alignment with an `internal` spec.  What do you think?

Reply:
Yeah ok, maybe the extra flexibility isn't needed at this time if we want to keep parity with Otel